includes/emails/email-summary/class-edd-email-summary.php

```diff
@@ -0,0 +1,348 @@
+				'exclude_taxes' => false,
+			)
+		);
+
+		$new_customers = $stats->get_customer_count();
+
+		$top_selling_products = $stats->get_most_valuable_order_items(
+			array(
+				'number' => 5,
+			)
+		);
+
+		$order_count = $stats->get_order_count();
+
+		return compact(
+			'earnings_gross',
+			'earnings_net',
+			'average_order_value',
+			'new_customers',
+			'top_selling_products',
+			'order_count'
+		);
+	}
+
+	/**
+	 * Generate HTML for relative markup.
+	 *
+	 * @since 3.1
+	 *
+	 * @param array $relative_data Calculated relative data.
+	 *
+	 * @return string HTML for relative markup.
+	 */
+	private function build_relative_markup( $relative_data ) {
+		$arrow  = $relative_data['positive_change'] ? 'icon-arrow-up.png' : 'icon-arrow-down.png';
+		$output = __( 'No data to compare', 'easy-digital-downloads' );
+		if ( $relative_data['no_change'] ) {
+			$output = __( 'No Change', 'easy-digital-downloads' );
+		} elseif ( $relative_data['comparable'] ) {
+			$output = $relative_data['formatted_percentage_change'] . '%';
+		}
+		ob_start();
+		?>
+		<?php if ( $relative_data['comparable'] ) : ?>
+			<img src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/icons/' . $arrow ); ?>" width="12" height="10" style="outline: none; text-decoration: none; -ms-interpolation-mode: bicubic; width: 12px; height: 10px; max-width: 100%; clear: both; vertical-align: text-top;">
+		<?php endif; ?>
+		<span style="padding-left: 1px;">
+			<?php echo esc_html( $output ); ?>
+		</span>
+		<?php
+
+		return ob_get_clean();
+	}
+
+	/**
+	 * Build email template.
+	 *
+	 * @since 3.1
+	 *
+	 * @param array|bool $blurb Structured blurb data.
+	 *
+	 * @return string|bool The string of the email template or false if the email template couldn't be built.
+	 */
+	public function build_email_template( $blurb = false ) {
+		$dataset = $this->get_report_dataset();
+		// If there were no sales, do not build an email template.
+		if ( empty( $dataset['order_count'] ) || 0 === $dataset['order_count'] ) {
+			return false;
+		}
+
+		$date_range     = $this->get_report_date_range();
+		$site_url       = $this->get_site_url();
+		$wp_date_format = get_option( 'date_format' );
+		$period_name    = ( 'monthly' === $this->email_options['email_summary_frequency'] ) ? __( 'month', 'easy-digital-downloads' ) : __( 'week', 'easy-digital-downloads' );
+		/* Translators: period name (e.g. week) */
+		$relative_text  = sprintf( __( 'vs previous %s', 'easy-digital-downloads' ), $period_name );
+
+		ob_start();
+		include EDD_PLUGIN_DIR . 'includes/emails/email-summary/edd-email-summary-template.php';
+
+		return ob_get_clean();
+	}
+
+	/**
+	 * Add tracking parameters to the blurb URL.
+	 *
+	 * @since 3.1
+	 *
+	 * @param  string $url Destination URL.
+	 * @param  array $blurb Blurb data.
+	 * @return string URL with tracking parameters.
+	 */
+	public function add_blurb_tracking_parameters( $url, $blurb ) {
+		// @todo - Define tracking parameters!
+		return add_query_arg(
```

Comment:
As we talked on the call yesterday, we can remove these, as we're going to supply these in the JSON data.